app/models/record/event_record.rb

```diff
@@ -31,7 +31,7 @@ def convert_xml_to_hash
 
   def parse_single_event_from_xml(event)
     event_data = {
-      id: event.at_xpath("E_ID").try(:text),
+      external_id: event.at_xpath("E_ID").try(:text),
       title: event.at_xpath("E_TITEL").try(:text),
       description: event.at_xpath("E_BESCHREIBUNG").try(:text),
       price_informations: [{ description: event.at_xpath("E_PREISTEXT").try(:text) }],
```

Comment:
@SoleCincis here we see also that events prices are only a description. nothin more is imported.